solr/core/src/test/org/apache/solr/search/TestPseudoReturnFields.java

```diff
@@ -126,7 +126,7 @@ public void testAllRealFields() throws Exception {
               ,"//result/doc/str[@name='ssto']"
               ,"//result/doc/str[@name='subject']"
               
-              ,"//result/doc[count(*)=4]"
+              ,"//result/doc[count(*)=5]"
```

Comment:
The numbers were changed since now _root_ is also added to the doc,
increasing the field count by 1.